container/images/base/fetch_repo_and_collect_coverage.sh

```diff
@@ -49,6 +49,13 @@ else
 fi
 # Output: Repo available at ${LOCAL_REPO_DESTINATION}
 
+# Convert from Dos to Unix - in case format is in non-unix format
+echo "Convert from Dos to Unix - in case format is in non-unix format"
+find ${LOCAL_REPO_DESTINATION} -type f           \
+     ! -path "*/.git/*"  \
+     ! -path "*/.idea/*" \
+     | xargs -n1 -I {} dos2unix {} || true
```

Comment:
If I understood correctly, we only need to apply dos2unix to text files. 
A better, more precise strategy to get the list of files to parse is the following:
- Initialise (or re-initialise) a git repo in destination folder `git init`
- Use the information in .gitignore to identify the relevant files `git ls-files`
- Filter known binary files such as `.jar`